src/directives.rs

```diff
@@ -118,7 +119,7 @@ pub fn extract_isort_directives(lxr: &[LexResult]) -> IsortDirectives {
                 off = None;
             }
         } else {
-            if comment_text.contains("isort: skip") {
+            if comment_text == "# isort: skip" {
```

Comment:
What about comments with multiple directives?

![Screen Shot 2023-01-07 at 12 18 50 PM](https://user-images.githubusercontent.com/1309177/211162733-4e1e34dc-fe0e-48a6-9554-a9b827b2c665.png)

Reply:
I see, then Let's revert this change.